projects/wipp-frontend-lib/src/lib/notebook/notebook-detail/notebook-detail.component.ts

```diff
@@ -21,6 +22,8 @@ import {HttpClient} from '@angular/common/http';
 import {NgxSpinnerService} from 'ngx-spinner';
 import {ModalErrorComponent} from '../../modal-error/modal-error.component';
 import {PlatformLocation} from '@angular/common';
+declare var nb: any;
+const marked = markedImported;
```

Comment:
why not just import as marked?

Reply:
Importing as marked caused an error "marked not found", this workaround got it to work.

Reply:
In that case, do we need to rename the variable? Can't you just markedImport directly?

Reply:
Renaming the variable doesn't solve the issue either. To be precise this is the error I get during the build:

Cannot call a namespace ('markedImported')
Error: Cannot call a namespace ('markedImported')

This issue seems to be related to the latest version of Angular.

Here's some few links with similar issues:
https://discourse.algolia.com/t/error-cannot-call-a-namespace-algoliasearch/6632
https://stackoverflow.com/questions/59735280/angular-8-moment-error-cannot-call-a-namespace-moment